searx/settings.yml

```diff
@@ -924,6 +924,12 @@ engines:
     shortcut: hex
     disabled: true
 
+  - name: crates.io
+    engine: crates
+    shortcut: crates
+    disabled: true
+    timeout: 6.0
```

Comment:
Forgot to say: I had timeout issues on search terms like `!cargo test` or `!cargo lib` .. so I increased timeout from the default 3sec to 6sec .. not sure this is a good value.